lib/github.js

```diff
@@ -0,0 +1,133 @@
+const { graphql } = require('@octokit/graphql')
+const { Octokit } = require('@octokit/rest')
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN
+})
+
+const graphqlWithAuth = graphql.defaults({
+  headers: {
+    authorization: `token ${process.env.GITHUB_TOKEN}`
+  }
+})
+
+module.exports.getPackageJson =
+async function getPackageJson (owner, repo) {
```

Comment:
```suggestion
module.exports.getPackageJson = async function getPackageJson (owner, repo) {
```
minor nit: it's awkward to have a line break after an `=`. same throughout.

Reply:
I specifically did not pick on this and other nits, as this should all be covered by the linters? I assume half of that is `standard`... But I don't use that style, so maybe I'm wrong. Regardless - we should just add an `.eslintrc`, discuss the actual style in a separate issue if we have to and be done with it, rather than pick on this in prototype PRs.

Reply:
That's generally fair, but I'm not sure there's a linter rule that properly blocks line breaks after `=`, which is why i commented on this one (and not on all the missing semicolons :-p)

Reply:
Fixed throughout